src/libraries/Native/AnyOS/System.IO.Compression.Native/entrypoints.c

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include <string.h>
+#include <stdint.h>
+
+// Include System.IO.Compression.Native headers
+#include "../zlib/pal_zlib.h"
+#include "../brotli/include/brotli/decode.h"
+#include "../brotli/include/brotli/encode.h"
+#include "../brotli/include/brotli/port.h"
+#include "../brotli/include/brotli/types.h"
+
+#ifndef lengthof
+#define lengthof(rg)    (int)(sizeof(rg)/sizeof(rg[0]))
+#endif
+
+typedef struct
+{
+    const char* name;
+    const void* method;
+} Entry;
+
+static Entry s_compressionNative[] =
+{
+    {"BrotliDecoderCreateInstance", (void*)BrotliDecoderCreateInstance},
```

Comment:
Can these be defined using macro to avoid mismatches between the string and the symbol name?

Reply:
Yes, I think these two will always match.